src/Google/GlobalSiteTag.php

```diff
@@ -172,4 +418,22 @@ public function maybe_display_event_snippet( string $ads_conversion_id, string $
 	public static function is_needed(): bool {
 		return true;
 	}
+
+	/**
+	 * Check if it is the new customer order.
+	 *
+	 * @param string $customer_email Customer email address.
+	 * @return bool True if this is new customer order.
+	 */
+	private static function is_first_time_customer( $customer_email ): bool {
+		$query = new \WC_Order_Query(
+			[
+				'limit'  => 2,
+				'return' => 'ids',
+			]
+		);
+		$query->set( 'customer', $customer_email );
+		$orders = $query->get_orders();
+		return count( $orders ) === 1 ? 'true' : 'false';
```

Comment:
The return type for this function is `bool` so best to return it as `true` or `false` without quotes.

Reply:
Updated.